docs/tutorial/usage.rst

```diff
@@ -45,6 +45,43 @@ interface, you'll see that a message has been sent to the ``amq.topic``
 exchange. However, since noone is listening to this topic, the message has been
 discarded. Now, we'll setup listeners.
 
+
+
+Consuming Messages
+------------------
+
+Consuming messages with a message schema ensures that messages adhere to predefined standards, facilitating interoperability and consistency within the Fedora Messaging ecosystem. This section provides guidance on how consumers can validate received messages against a schema and convert the base `Message` into more specific objects based on the schema definition.
+
+Validating Messages Against Schema
+----------------------------------
+
+Before processing a received message, consumers must validate it against the associated message schema. This ensures that the message structure conforms to the expected format defined in the schema. The `jsonschema` library is commonly used for this purpose within the Fedora Messaging project.
```

Comment:
This is actually not true, the message is already validated as part of the consuming process (in `message.get_message()`.

Reply:
So I should remove this part? any suggestions?

Reply:
Yeah, but that means removing everything actually, because the introduction in "Consuming Messages" is wrong too: it mentions how consumers can validate received messages, and they don't need to.